docs/user-guide/ze-use-cases.md

```diff
@@ -307,3 +307,70 @@ The following use cases demonstrate the various functionalities available in Zow
    The output displays in the **Output** panel.
 
    ![Issue a TSO command](../images/ze/ZE-TSO-Command.gif)
+
+### Polling a spool file
+
+Users can periodically refresh a spool file during long-running jobs to get the latest job outputs. This avoids having to close and reopen a spool file to get the latest updates.
```

Comment:
Thoughts on using "these updates" instead of "the latest updates"? That way it refers to the "latest job outputs" in the previous sentence?

Reply:
done -- thank u